microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -106,41 +119,42 @@ The following are some of the flags we've seen used.
 | `js-flags` | Specifies the flags passed to the JS engine.  Available flags: `scavenger_max_new_space_capacity_mb`: Specifies the maximum limit (in MB) for scavenger (minor) garbage collectors in the V8 JavaScript engine.  For example: <br/>`--js-flags=--scavenger_max_new_space_capacity_mb=8`  <br/>A lower scavenger memory limit reduces memory usage, and increases the frequency of running minor garbage collectors.  <br/>A higher scavenger memory limit increases memory usage, and reduces the frequency of running minor garbage collectors. |
 | `lang` | The language file that WebView2 want to try to open. Of the form language[-country] where language is the 2-letter code from ISO-639. |
 | `log-net-log` | Enables saving net log events to a file.  If a value is given, that value is used as the directory path and file name.  If no value is given, the file is named `netlog.json`, and is placed in the user data directory. |
+| `long-animation-frame-timing` |  Provides detailed timing for long animation frames, which are frames exceeding the standard 16.67ms duration (60fps).  This helps during debugging and performance analysis, to identify and optimize performance bottlenecks for animation-heavy applications. |
```

Comment:
* During development, what is the result of this `long-animation-frame-timing` flag being present, in contrast to the norm which is that this flag is not present?  That might be answered by the existing text.
* Does this flag take a true|false value, or some other value?
* Is there an example of this flag in use?
@sivMSFT @captainbrosset

Reply:
didn't get info about long-animation-frame-timing.  closing comment b/c plan to replace the PR